### DISCUSSION TITLE
Error in Phi4 example

### DISCUSSION THREAD

**Discussion by feyncalccc** [FeynCalc/feyncalc#220] [Q&A] [answered]
Hi all! I'm new to FeynCalc and am currently running through some examples coming with the package. I consistently got errors when the models in the examples are generated by FeynRules, but not when the models are included in the FeynCalc package. For example, in the Phi4 One-loop example, I got something like this in the attached picture. I am thinking this may be due to some incompatibility between FeynRules and Mathematica. I really appreciate it if anyone could kindly remind me of what is going on<img width="779" alt="Screen Shot 2023-05-27 at 3 20 18 PM" src="https://github.com/FeynCalc/feyncalc/assets/134809115/20fe188b-5323-4dbb-ba5e-93b9ccf82b29">

**Comment by vsht** [comment, marked as answer]
It seems like you are trying to run both scripts on the same kernel, which is a no-go. You must first run
`GenerateModelPhi4.m`, then quit Mathematica completely (including **all** kernels) and only then run the example file
on a fresh **kernel**.

**Reply by feyncalccc** [reply]
Thanks vey much! that is exactly the issue.
